config/environment.js

```diff
@@ -43,9 +40,6 @@ module.exports = function (environment) {
   if (environment === 'test') {
     // Testem prefers this...
     ENV.locationType = 'none';
```

Comment:
The below credentials could be mishandled, ensure it doesn't get merged in general

Reply:
I have removed this credential from this PR. It was needed before to make the 100ms APIs work. Now we are using our own backend API so, it is not needed. For security purposes, I have disabled the room ID.